### PR TITLE
Update simulator related wording to send D2C

### PR DESCRIPTION
**Pull request by blackchoey**
1. Update simulator related wording to send D2C
2. Change urls in package-lock.json to https
3. Update send D2C message part in welcome page. The gif in welcome page is hosted in github wiki, we need to update the gif along with the new version release.

### CODE REVIEW

File: src/simulator.ts [hunk 1]
```diff
@@ -210,7 +210,7 @@ export class Simulator {
       this.cancelToken = false;
     } else {
       vscode.window.showErrorMessage(
-        "A previous simulation is in progress, please wait or cancel it.",
+        "A previous send D2C messages operation is in progress, please wait or cancel it.",
```
Comment (formulahendry) [inline src/simulator.ts]: send -> sending?

File: src/simulator.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export class Simulator {
   public static getInstance(context?: vscode.ExtensionContext) {
     if (!Simulator.instance) {
       if (!context) {
-        vscode.window.showErrorMessage("Cannot create Simulator instance.");
+        vscode.window.showErrorMessage("Cannot create Send D2C Messages webview instance.");
```
Comment (adashen) [inline src/simulator.ts]: remove "instance"?

File: src/simulator.ts [hunk 3]
```diff
@@ -155,19 +155,19 @@ export class Simulator {
       } else {
         // Exit when no connection string found or the connection string is invalid.
         if (!this.iotHubConnectionString) {
-          vscode.window.showErrorMessage("Failed to launch Simulator: No IoT Connection String Found.");
+          vscode.window.showErrorMessage("Failed to launch Send D2C Messages webview: No IoT Connection String Found.");
```
Comment (adashen) [inline src/simulator.ts]: IoT Hub Connection String?

File: src/simulator.ts [hunk 4]
```diff
@@ -155,19 +155,19 @@ export class Simulator {
...
           this.telemetry(Constants.SimulatorLaunchEvent, false, {
-            error: "Failed to launch Simulator: No IoT Connection String Found.",
+            error: "Failed to launch Send D2C Messages webview: No IoT Connection String Found.",
```
Comment (adashen) [inline src/simulator.ts]: same as above, can we has a variable for the message?

File: src/simulator.ts (hunk 2 above)
Comment (blackchoey) [reply src/simulator.ts]: Actually it's not creating the webview. I will update the error to: Cannot initialize Send D2C Messages webview resources.

**Pull request merged**